openshift/templates/nginx.json

```diff
@@ -16,7 +16,8 @@
   },
   "message": "The following service(s) have been created in your project: ${NAME}.\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/sclorg/nginx-ex/blob/master/README.md.",
   "labels": {
-    "template": "nginx-example"
+    "template": "${NAME}",
```

Comment:
How do we ensure that `$NAME` => `nginx-example`?

Reply:
@ardaguclu By parameter -p NAME when executing `oc new-app`.

Reply:
But I will clarify it in OpenShift 4 environment.

Reply:
Yeah it works. In template yaml I see in labels `name: nginx-testing`